apis/Google.Cloud.Diagnostics.AspNetCore/docs/index.md

```diff
@@ -246,3 +246,34 @@ that is propagated in a `custom_trace_id` header. This is of no use in the real
 
 {{sample:Trace.CustomTraceContext}}
 
+## Using Google Trace in other than ASP.NET Core applications
+
+If you want to use Google Cloud Trace in applications that are not ASP.NET Core you may use the
+`Google.Cloud.Diagnostics.Common` package directly and .NET's dependency injection mechanism.
+You can read more about .NET dependency injection in non ASP.NET Core apps
+[here](https://docs.microsoft.com/en-us/dotnet/core/extensions/dependency-injection-usage).
+
+Note that this is useful for both installed applications and services that process incoming messages
+other than HTTP requests, for instance, a service that it's subscribed to Pub/Sub topics.
```

Comment:
it's => is? Or perhaps replace everything from "for instance" with "such as a service reacting to Pub/Sub messages"?